app/models/venue.rb

```diff
@@ -18,6 +18,16 @@ class Venue < ActiveRecord::Base
   has_many :reviews
   belongs_to :user
 
+  include PgSearch
+  pg_search_scope :search_by_venue_name,
+  against: :venue_name,
+  :using => {
+              :tsearch => {:prefix => true}
+            }
```

Comment:
Indent the right brace the same as the start of the line where the left brace is.